### PR TITLE
Refactoring fragment tags globally and config/env.js paths

### LINKED ISSUE #40115: Replace <Fragment> tags to <> around the projects
**Describe the bug**
There are many <Fragment></Fragment> tags around the codebase. Please replace it to <></> in order to get a more concise and clear code.

fragment

### PR DESCRIPTION
**Pull request by evertonrbraga**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40115

### CODE REVIEW

File: client/src/components/profile/components/Certifications.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const propTypes = {
 
 function renderCertShow(username, cert) {
   return cert.show ? (
-    <Fragment key={cert.title}>
+    <>
```
Comment (Sky020) [inline client/src/components/profile/components/Certifications.js:41]: The shorthand cannot replace the Fragment with the `key` property.

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ In this challenge, you will create a utility for your tree. Write a JavaScript m
 ```yml
 tests:
   - text: Your Binary Search Tree should return true when checked with <code>isBinarySearchTree()</code>.
-    testString: assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; test.push(3); test.push(4); test.push(5); return isBinarySearchTree(test) == true})());
+    testString: assert((function() { var test = false; if (typeof BinarySearchTree !== 'undefined') { test = new BinarySearchTree() } else { return false; }; test.push(1); test.push(5); test.push(3); test.push(2); test.push(4); return isBinarySearchTree(test) == true})());
```
Comment (Sky020) [inline curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md:28]: Why is this in this PR?

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
I am not sure about the unrelated changes to the `env`, as well as the one test change.

File: client/src/components/profile/components/Certifications.js (hunk 1 above)
Comment (evertonrbraga) [reply client/src/components/profile/components/Certifications.js]: Fixed

File: curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md (hunk 2 above)
Comment (evertonrbraga) [reply curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md:28]: I don't know, maybe I commited some mistakes using VSCode shortcuts. But I recovered it.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Hi @evertonrbraga, thanks for this PR.  However, it should be restricted to changing only the fragment tags, since the other changes break the build. Please could you revert them?

**Comment by evertonrbraga** [comment]
@ojeytonwilliams it's done :)

**Comment by Sky020** [comment]
@evertonrbraga It does not seem like you have your linter set up with the fCC styles. This is causing the tests to fail. I recommend going over the docs on this: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=how-to-prepare-your-local-machine

---

Also, I am not sure where the discussion happened to make these changes?

**Comment by evertonrbraga** [comment]
@Sky020 I'm gonna read the contribution guidelines carefully. I'm starting at the open source world and I'm learning yet. I saw an opportunity of contribution by refactoring the Fragment tags, but I got some problems. I'm sorry for it, and if you think is better to discard this PR, go on. I'll try to understand it deeply before contribute again. Thank you!

**Comment by Sky020** [comment]
@evertonrbraga There is no need to discard anything. It would just be more effective to discuss some changes, before implementing them.

**Comment by ojeytonwilliams** [comment]
Hey @evertonrbraga, did you get your setup working?  This just needs the merge conflict and lint errors fixing and it's good to merge.

**Comment by evertonrbraga** [comment]
Hey @ojeytonwilliams, what's up? I only have to config my eslint to get success? Sorry my delay, I'm a little busy, but I intend to do this weekend, okay? Thank you!

**Comment by raisedadead** [comment]
Closing this stale for now, for a head branch rename. Thanks for your PR.

**Pull request closed without merging**